deploy/crds/kieapp.crd.yaml

```diff
@@ -97,45 +93,261 @@ spec:
                 server:
                   type: object
                   description: Configuration of the RHPAM KIE server
-#                  anyOf:
-#                  - required:
-#                    - env
-#                  - required:
-#                    - resources
                   properties:
-                    env:
-                      type: array
-                      items:
+                    deployments:
+                      type: integer
+                      format: int16
+                      description: Number of Server sets that will be deployed
+                    from:
```

Comment:
Is this optional, and does it default to "from RH kie server images"?

Reply:
Yes it is optional and defaults to RH Kies Server images. See 
https://github.com/kiegroup/kie-cloud-operator/pull/63/commits/a0e904a0a46665dcde97eb9aa82eaed262f0cdfb#diff-53394291f02ee59483694ad21df717b5R167-R177